distribution/config/supervisord.conf

```diff
@@ -18,7 +18,7 @@ startretries=20
 user=jenkins
 
 [program:jenkins]
-command=/usr/local/bin/jenkins-evergreen.sh
+command=/evergreen/scripts/jenkins-evergreen.sh
```

Comment:
Followup NIT, I think it should be `%(ENV_EVERGREEN_HOME)s/scripts/jenkins-evergreen.sh`

but :+1: indeed for moving all the things into the writable /evergreen root overall